window.js

```diff
@@ -2229,6 +2229,10 @@ var WindowManager = GObject.registerClass(
         resizePairForWindow = this.tree.nextVisible(focusNodeWindow, direction);
       }
 
+      if (this.floatingWindow(focusNodeWindow)) {
```

Comment:
You can use `focusNodeWindow.isFloat()` as an alternative syntax and you can add this if condition at line 2209 that it becomes below:
```
if (!focusNodeWindow || focusNodeWindow.isFloat()) return;
```
This saves some cycles of not calculating grab etc.